swig/CMakeLists.txt

```diff
@@ -24,9 +24,9 @@ else (WIN32)
 endif (WIN32)
 
 if (CMAKE_MAJOR_VERSION LESS 3)
-  set_source_files_properties(${HOTROD_SWIG_MODULE_SOURCE} PROPERTIES SWIG_FLAGS "-v;-namespace;Infinispan.HotRod.SWIGGen;-dllimport;${HOTROD_WRAP_LIB}")
+  set_source_files_properties(${HOTROD_SWIG_MODULE_SOURCE} PROPERTIES SWIG_FLAGS "-DSWIG2_CSHARP;-v;-namespace;Infinispan.HotRod.SWIGGen;-dllimport;${HOTROD_WRAP_LIB}")
 else (CMAKE_MAJOR_VERSION LESS 3)
-  set_source_files_properties(${HOTROD_SWIG_MODULE_SOURCE} PROPERTIES SWIG_FLAGS "-v;-namespace;Infinispan.HotRod.SWIGGen;")
+	set_source_files_properties(${HOTROD_SWIG_MODULE_SOURCE} PROPERTIES SWIG_FLAGS "-DSWIG2_CSHARP;-v;-namespace;Infinispan.HotRod.SWIGGen;")
```

Comment:
Indentation seems wrong here?

Reply:
vim on win makes me crazy :) should be ok now @alanfx

Reply:
That would drive me crazy too, @rigazilla! Looks good now.